src/components/rangeFilterInput/index.stories.mdx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { Box } from '@chakra-ui/react';
+
+import RangeFilterInput from './index';
+
+<Meta title="Components/Forms/FilterInput" component={RangeFilterInput} />;
+
+export const Template = ({ name, placeholder, size, unit, value, ...rest }) => {
+  const [currentValue, setCurrentValue] = useState(value);
+  const onChange = (editedField, newValue) => {
+    setCurrentValue({
+      ...currentValue,
+      [editedField]: newValue[editedField],
+      touched: editedField,
+    });
+  };
+  return (
+    <Box maxW={270}>
+      <RangeFilterInput
+        name={name}
+        value={value}
+        placeholder={placeholder}
+        handleChange={(event) => {
```

Comment:
The stories for `Input` component leverage actions addon to show the events in the actions tab. Any change you could add it here too?

Reply:
Ping about the actions